File: azure/transporterSolution/transporterService/DataObjects/OrderStatusItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class OrderStatusItem : BaseItem
    {
        public OrderStatusItem()
        {
            ImageItems = new List<ImageItem>();
        }

        public bool IsActual { get; set; }
        public string DateTime { get; set; }
        public virtual OrderItem OrderItem { get; set; }
        public virtual StatusDescriptionItem StatusItem { get; set; }

        public virtual ICollection<ImageItem> ImageItems { get; set; }
        
    }
}

File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/OrderStatusItem.java
package com.example.fugliara.ccs002.server.dataobjects;

import java.util.List;

public class OrderStatusItem {
    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("isActual")
    public boolean isActual;
    @com.google.gson.annotations.SerializedName("dateTime")
    public String dateTime;
    @com.google.gson.annotations.SerializedName("orderItem")
    public OrderItem orderItem;
    @com.google.gson.annotations.SerializedName("statusItem")
    public StatusDescriptionItem statusItem;
    @com.google.gson.annotations.SerializedName("imageItems")
    public List<ImageItem> imageItems;

    public OrderStatusItem() {
        id = "";
        isActual = false;
        dateTime = "";
        orderItem = null;
        statusItem = null;
        imageItems = null;
    }
}
